Prog/2019/Internet Access.sas

```diff
@@ -0,0 +1,183 @@
+           Numsatelliteonly_&year. Numotheronly_&year. Numaccesswosub_&year. Numnointernet_&year. tothhdemom pctinternet percent100Kplus percentunder15K
+           percent15to50K percent50to75K percent75Kto100K county metro15 Jurisdiction hshldinc100000plus_2013_17 hshldincunder15000_2013_17 hshldinc15000to34999_2013_17 hshldinc35000to49999_2013_17
+           hshldinc50000to74999_2013_17 hshldinc75000to99999_2013_17 pctinternet pctnointernet Numwithinternet_2013_17;
+
+      county= substr(geo2010, 1,5);
+	  ucounty=county;
+	  geoid= geo2010;
+
+%ucounty_jurisdiction
+
+	  metro15 = put( county, $ctym15f. );
+      tothhdemom= hshldinc100000plus_2013_17+ hshldincunder15000_2013_17+ hshldinc15000to34999_2013_17+ hshldinc35000to49999_2013_17+
+      hshldinc50000to74999_2013_17+ hshldinc75000to99999_2013_17;
+      percent100Kplus= hshldinc100000plus_2013_17/tothhdemom;
+	  percentunder15K= hshldincunder15000_2013_17/tothhdemom;
+	  percent15to50K= (hshldinc15000to34999_2013_17+ hshldinc35000to49999_2013_17)/tothhdemom;
+	  percent50to75K= hshldinc50000to74999_2013_17/tothhdemom;
+      percent75Kto100K= hshldinc75000to99999_2013_17/tothhdemom;
+	  internetdenom= Numdialup_&year. + Numbroadbandall_&year. + Numcellular_&year. + Numcellularonly_&year. + Numbroadband_&year. + Numbroadbandonly_&year. + Numsatellite_&year.+ 
+                     Numsatelliteonly_&year. + Numotheronly_&year. + Numaccesswosub_&year. + Numnointernet_&year.;
+	  pctinternet= (internetdenom- Numnointernet_&year.)/internetdenom;
+	  pctnointernet= 1- pctinternet;
+
+	  format Jurisdiction Jurisdiction. ;
+run;
+
+proc export data= InternetAcccess
+   outfile="&_dcdata_default_path\Requests\Prog\2019\Internet_access_tract.csv"
+   dbms=csv
+   replace;
+run;
+
+data BroadbandAcccess ;
+	  set acs.Acs_2013_17_va_sum_tr_tr10 acs.Acs_2013_17_dc_sum_tr_tr10 acs.Acs_2013_17_md_sum_tr_tr10 acs.Acs_2013_17_wv_sum_tr_tr10;
+	      
+	  keep geo2010 geoid totpop_&year. medfamincm_&year. popalonew_&year. popaloneb_&year. popaloneh_&year. popalonea_&year. popaloneaiom_&year. numpopbroadbandtot
+	       numpopbroadbanda_&year. numpopbroadbandiom_&year. numpopbroadbandb_&year. numpopbroadbandh_&year. numpopbroadbandw_&year.
+	       hshldinc100000plus_2013_17 hshldincunder15000_2013_17 hshldinc15000to34999_2013_17 hshldinc35000to49999_2013_17
+           hshldinc50000to74999_2013_17 hshldinc75000to99999_2013_17 percent100Kplus percentunder15K percent15to50K percent50to75K  percent75Kto100K
+           pctbroadband pctbroadbanda pctbroadbandb pctbroadbandh pctbroadbandw pctbroadbandiom  Numnointernet_2013_17 pctinternet pctnointernetcounty metro15 Jurisdiction Numbroadband_2013_17 Numhhdefined_2013_17;
+
+      county= substr(geo2010, 1,5);
+	  ucounty=county;
+	  geoid= geo2010;
+
+%ucounty_jurisdiction
+
+	  metro15 = put( county, $ctym15f. );
+      pctbroadband= Numbroadband_2013_17/totpop_2013_17;
+	  pctbroadbanda= numpopbroadbanda_&year./popalonea_&year.;
```

Comment:
poplalonea is for all people not just people in households. to get the right denominator you need to add NumPopdialupA_2013_17 + NumPopbroadbandA_2013_17 + NumPopnointernetA_2013_17 + NumPopnocomputerA_2013_17 since you didn't read in the total cell for that table. Please adjust for all the by race indicators.